docs/root/call-settings.md

```diff
@@ -58,3 +58,74 @@ for multiple RPC invocations.
+absolute, e.g. "it must finish by 2016-12-07 14:55:00Z"). You create an `Expiration` by specifying that timeout
+or deadline, with the `Expiration.FromTimeout(TimeSpan)` and `Expiration.FromDeadline(DateTime)` methods.
+Alternatively, you can use `Expiration.None`, which is an infinite timeout. A timeout is always relative to the
+start of an operation, not the time the expiration is created. So for example, if you create an expiration of
+5 seconds, you can use that as often as you like, and it will always mean that the operation you're applying
+the expiration to has 5 seconds to complete.
+
+Note that deadline expirations are most commonly used when the same deadline is passed to multiple calls:
+if you have to perform 5 different RPCs, and you know your total work needs to finish at a particular
+point but you don't care how those 5 RPCs use up your time budget between them, passing in the same
+deadline to all 5 works well. This is restricted to one unit of work, however - you'd need to calculate a new
+deadline for the next unit of work, whereas a relative timeout may be valid for all calls. You should think
+carefully about your own requirements - and please file a feature request if we don't provide enough functionality
+for you to meet them.
+
+### RetrySettings
+
+Retry is inherently complicated. There will be potentially multiple RPCs made, each with its own expiration, some
+back-off between RPCs, and the situations under which it's reasonable to retry. The [RetrySettings](obj/api/Google.Api.Gax.Grpc.RetrySettings.yml) exposes this via the following properties (all set by the constructor):
+
+- `TimeoutBackoff`: how the expiration for each individual RPC varies
+  (for example, the first RPC might have a timeout
+  of 1s, then the second RPC might have 2s etc)
+- `RetryBackoff`: how the *delay* between each failed RPC varies
+  (for example, we might wait for 0.5s after the first RPC, then 0.75s after the second, etc)
+- `RetryFilter`: a predicate which is consulted after each failed RPC to see whether that failure
+  suggests we should retry. For example, we might retry on timeouts but not on a "resource not found" failure.
+- `TotalExpiration`: the overall expiration for the whole operation. For example, we might allow multiple RPCs
+  to attempt an operation, but the whole operation must have completed within 10 seconds or we'll fail with a timeout.
+- `DelayJitter`: This allows the precise amount of delay (broadly specified by `RetryBackoff`) to vary randomly
+  a little. This avoids multiple clients accidentally synchronizing their retries and periodically hitting
+  a service very hard. This is exposed for testability, but is unlikely to need tweaking in production use.
+
+ `TimeoutBackoff` and `RetryBackoff` are both expressed in terms of a [BackoffSettings](obj/api/Google.Api.Gax.Grpc.BackoffSettings.yml), which consists of an initial timeout, a maximum timeout, and a multiplier.
```

Comment:
This is a wall of text that I hesitate to read.  What's more, is that it's a list of individual definitions, where I need to know how all these parameters interact.

Again, I think the way that will most directly communicate how all these variables interact is to show me the retry loop, in code or pseudocode.